generate/generate.go

```diff
@@ -657,6 +733,10 @@ func (g *Generator) convertEnum(tspec *ast.TypeSpec) (*desc.EnumDescriptorProto,
 		}
 	}
 	g.enumIndex++
+	// If an enum doesn't have any values
+	if len(enum.Value) == 0 {
```

Comment:
This is required because protoc is more strict that protoc-gen-go. protoc won't compile the proto if there is "hanging" enums. We should either error out if there is no enum values, or ignore that enum. This is currently happening due to the `type`s in https://github.com/gunk/opt/blob/master/opt.gunk

Reply:
SGTM; this was an oversight on my part anyway.